### PR TITLE
Quick update

### PR DESCRIPTION
**Pull request by suirad**
Made the Venus binding of Mercury handle messages from plugins in the same order as the other bindings.

Pull for NameLayer to reflect that as well as merging my repo with civraft/namelayer:master

Only issue since i have merged is i now cannot get the plugin to load on my test server, see below image:
img

### CODE REVIEW

**Comment by rourke750** [comment]
You need to build using maven.

**Comment by suirad** [comment]
I am supposed to use "Run As > Maven install" correct?
Using that i got this:
```
Failed to execute goal on project NameLayerNMS: Could not resolve dependencies for project vg.civcraft.mc.namelayernms:NameLayerNMS:jar:1.0.0: The following artifacts could not be resolved: org.bukkit:craftbukkit:jar:1.7.10-R0.1, org.spigotmc:spigot:jar:1.8-R0.1, org.spigotmc:Spigot1.8.3:jar:1.8.3, org.spigotmc:Spigot1.8.7:jar:1.8.7: Failure to find org.bukkit:craftbukkit:jar:1.7.10-R0.1 in http://civcraft.co:8080/plugin/repository/everything/ was cached in the local repository, resolution will not be reattempted until the update interval of Jenkins-repo has elapsed or updates are forced -> [Help 1]
```

**Comment by rourke750** [comment]
Uh I did everything today let me fix the pom.

**Comment by rourke750** [comment]
Try now.

**Comment by suirad** [comment]
All good on this end for a merge.

**Comment by rourke750** [comment]
remove this line: https://github.com/Civcraft/NameLayer/blob/master/plugin.yml#L4 then remove the delay.

**Comment by suirad** [comment]
Done, see last commit

File: src/vg/civcraft/mc/namelayer/NameLayerPlugin.java [hunk 1]
```diff
@@ -60,8 +59,9 @@ public void onLoad(){
 	public void registerListeners(){
 		getServer().getPluginManager().registerEvents(new AssociationListener(), this);
 		getServer().getPluginManager().registerEvents(new PlayerListener(), this);
-		if (isMercuryEnabled){
-			getServer().getPluginManager().registerEvents(new MercuryMessageListener(), this);
+		if (isMercuryEnabled()){
+			NameLayerPlugin nl = NameLayerPlugin.getInstance();
+			nl.getServer().getPluginManager().registerEvents(new MercuryMessageListener(), nl);
```
Comment (rourke750) [inline src/vg/civcraft/mc/namelayer/NameLayerPlugin.java]: replace nl with this

Comment (suirad) [inline src/vg/civcraft/mc/namelayer/NameLayerPlugin.java]: Already done. see last quick commit.

**Comment by rourke750** [comment]
@suirad Also do you know how to squash commits?

**Comment by suirad** [comment]
I do not. Educate me.

**Comment by rourke750** [comment]
Alright so the idea is that we merge multiple commits into one to not pollute the git history (I'm so guilty of this).  Look at this to try and do it: http://stackoverflow.com/questions/14534397/squash-all-my-commits-into-one-for-github-pull-request  and if that doesn't really show then just comment here again and I'll help you out.

**Comment by ProgrammerDan** [comment]
Rebasing against master when master has very far diverged from your branch is practically impossible without squashing, if your branch has many commits. I highly recommend learning this skill :)

Also for small changes `git commit --amend` is your friend. Just be careful, all history rewriting can be dangerous and I recommend doing it in temporary branches until you have mastery.

**Comment by jjj5311** [comment]
If someone comes up with a good way to squash let me know.... i basically
have to look at a tutorial every time i try to do it

**Comment by suirad** [comment]
@rourke750 which commit do you want me to squash to?

**Comment by rourke750** [comment]
@jjj5311 Same but it isnt too difficult.  I finally get how to fix merge conflicts without googling.

**Comment by rourke750** [comment]
@suirad All of them into one.

**Comment by suirad** [comment]
I think i got it. I could only go back to where i merged your changes from earlier today. But it was relatively easy to do. I could share if anyone is interested.

**Comment by suirad** [comment]
Good for merge?

**Pull request merged**